MaintenanceExam/TestCases/InsuranceQuote/RequestAutoPersonalInsuranceQuote.cs

```diff
@@ -18,6 +18,12 @@ public void RequestAutoPersonalInsuranceQuoteCase()
             Driver.Value!.Url = "https://www-qa.mybranchqa.org/insurance/personal/request-a-quote";
 
             new GetAQuotePage(Driver.Value!).CreateRequest(true, false);
+
+            GetAQuotePage getAQuotePage = new GetAQuotePage(Driver.Value!);
+
+            string confirmationMessageResults = getAQuotePage.ConfirmationMessage();
+
+            Assert.Pass(confirmationMessageResults, "Confirmation Message does display.");
```

Comment:
I can't remember if we've talked about this before or not? When you are comparing two strings, always use `Assert.AreEqual(expectedString, actualString, "assert descriptor goes here")` because when it fails, in the error message it prints both the actual and expected strings instead of "Expected True was False" (which isn't terribly helpful in narrowing down the issue). Also, I think you meant to use `Assert.True()?` and not `Assert.Pass()` which always passes (it's in the name :)).